### DISCUSSION TITLE
IKEA unsupported devices

### DISCUSSION THREAD

**Discussion by Suovenir** [Koenkk/zigbee2mqtt#22473] [Devices]
Hi, 
I just bought some Ikea TRADFRI things, but I have some issues...

the model are (all present in the compatible devices):

- PARASOLL door/window sensor - model E2013
- TRETAKT smart plug - model E2204
- RODRET wireless dimmer/power switch - model E2201

![image](https://github.com/Koenkk/zigbee2mqtt/assets/73658822/882b8576-85b4-49bc-9f39-24fedc2d7164)

this is the log:
```
Warning Received message from unsupported device with Zigbee model 'PARASOLL Door/Window Sensor' and manufacturer name 'IKEA of Sweden'
Warning Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.
Warning Received message from unsupported device with Zigbee model 'TRETAKT Smart plug' and manufacturer name 'IKEA of Sweden'
Warning Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.
Warning Received message from unsupported device with Zigbee model 'RODRET Dimmer' and manufacturer name 'IKEA of Sweden'
Warning Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.
```
I tried to pair multiple time, but I have the same error every time. It can depend on the name of the manufacturer "IKEA of Sweden" instead "IKEA" (I have some old light bulb with only "IKEA" and they work)?

Thanks in advance,
Cheers

**Comment by francisp2** [comment]
What version of Zigbee2MQTT is installed ?

**Reply by Suovenir** [reply]
I have the 1.18.1 version

Coordinator: ConBee2


![IMG_20240506_093621.jpg](https://github.com/Koenkk/zigbee2mqtt/assets/73658822/43b2888a-33cd-4dae-9bf0-3e793c78a6df)

**Reply by Suovenir** [reply]
Oh I see... My bad, ~but I don't have any updates from the home assistant page. How can I I solve this?~ I reinstalled the add-on following the guide and now everything works fine. Thank you very much for the suggestion.

**Reply by best-74** [reply]
Did you do anything other to get this fixed.. i have an Ikea Parasoll which just shows as unsupported, i am on the latest version of z2mqtt..

**Reply by Suovenir** [reply]
no, just this. to be more precise in my case i still had z2m installed with the old repo. therefore i can advise you to see if this is also your case (my last version was 1.18.1 as you can see from the screenshots, and it does not match with the latest version in the new repo) in case, you have to follow the guide for the installation with the new repo. i did not need to redo the pairings. i hope you solve it this way.
